hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -322,9 +322,9 @@
     {
       // Reused from hw/dv/tools/dvsim/tests/csr_tests.hjson.
       name: "chip_csr_bit_bash"
-      // Don't test over 512 randomly picked CSRs at a time.
-      run_opts: ["+test_timeout_ns=120_000_000", "+num_test_csrs=400"]
-      run_timeout_mins: 120
+      // Don't test over 200 randomly picked CSRs at a time.
```

Comment:
just curious, is 512 vs 200 mostly a run time thing?

Reply:
Note that 512 was a typo (it was set to 400). With `"+num_test_csrs=400"`, the failing seed in #15898 passed after 5 hours (way too long for this type of test). So we instead pick only 200 at a time. We reseed this test 5 times, which is not sufficient to cover the full spectrum of our CSR space in a single night's regression, so I will think about this a bit more.